packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -1242,6 +1242,15 @@ class ListValueNotifier<T> extends ChangeNotifier
     _listChanged();
     return true;
   }
+
+  /// Removes a range of elements from the list.
+  ///
+  /// https://api.flutter.dev/flutter/dart-core/List/removeRange.html
+  void removeRange(int start, int end) {
```

Comment:
can we add a test for this in the existing group of ListValueNotifier tests in utils_test.dart?